### PR TITLE
Add discovery for APC EPDU1132M

### PR DESCRIPTION
**Pull request by duhow**
This adds discovery to APC Easy Rack PDU model EPDU1132M.

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by duhow** [comment]
I'm unable to add the device sensors, it doesn't seem to be an existing MIB with these OIDs (which is weird), and have spent several hours on editing the YAML definition discovery without success.
At least this will provide better discovery than generic.

File: includes/definitions/apc-epdu.yaml [hunk 1]
```diff
@@ -10,3 +10,9 @@ over:
 discovery:
     - sysObjectID:
         - .1.3.6.1.4.1.318.1.3.4.9 
+    - sysObjectID:
+        - .1.3.6.1.4.1.0
```
Comment (Jellyfrog) [inline includes/definitions/apc-epdu.yaml:14]: Hmmm okey so this pdu uses a oid that belongs to some trap mib.. .great... would you mind investigate a bit, maybe upgrade firmware if possible? see if they fixed something?

Comment (duhow) [reply includes/definitions/apc-epdu.yaml:14]: I found this `EPDU_MP1_VM_V2813.zip` update, but after reaching technical service, they told me this update was for model EPDU1116M , so it appears to be no other public firmware upgrade.

**Comment by Jellyfrog** [comment]
We're missing the test data json file for 1132M

**Comment by duhow** [comment]
Done @Jellyfrog

**Review by Jellyfrog — APPROVED** [review]

**Pull request merged**